public/i18n/module/i18n_module_en.properties

```diff
@@ -22,3 +22,12 @@ abort=abort
 
```

Comment:
Take a look at the [d2-i18n library](https://github.com/dhis2/d2-i18n#table-of-contents) we've standardised on for DHIS2 apps.

Reply:
Yes, I'll look into it during the next sprint.